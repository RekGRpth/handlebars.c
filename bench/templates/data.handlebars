{{#each names}}{{@index}}{{name}}{{/each}}