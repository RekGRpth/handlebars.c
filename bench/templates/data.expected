0Moe1Larry2Curly3Shemp
